app/cosmos/databases/applications.js

```diff
@@ -0,0 +1,21 @@
+const cosmosClient = require('../client')
+const { cosmosConfig } = require('../../config')
+
+const applicationsDatabase = async () => {
+  try {
+    const { database } = await cosmosClient.databases.createIfNotExists({
+      id: cosmosConfig.applicationsDatabase,
+      throughput: 400
```

Comment:
More a question, what is `throughput` for and why did we decide `400`?

Reply:
`throughput` is the amount of resource we allocate to a specific databases container. 

400 was the default value 😝 

Its an area we need to look more into although i think the conversation with ADP will shed some more light on it. There are options to set finite resource vs autoscale to demand if we had one databases container that was seeing a larger quantity of requests.

https://learn.microsoft.com/en-us/azure/cosmos-db/set-throughput